contentstore/models.py

```diff
@@ -78,6 +78,8 @@ class MessageSet(models.Model):
 
     short_name = models.CharField(_('Short name'), max_length=100, unique=True)
     notes = models.TextField(_('Notes'), null=True, blank=True)
+    channel = models.CharField(_('Channel'), max_length=64, null=True,
```

Comment:
There is going to be a channel model in the message sender, this string is sent to the message sender with the outbound payload to select the correct channel. Not ideal but better than managing the channels in two places I think.